### DISCUSSION TITLE
TIF to PDF conversion results in high brightness

### DISCUSSION THREAD

**Discussion by tomasr78** [ImageMagick/ImageMagick#5138] [Help] [answered]
I have noticed that converting TIF files to PDF results in a high brightness-washed outlook. See the example here 
[test.zip](https://github.com/ImageMagick/ImageMagick/files/8650130/test.zip)

![167092987-65a84a32-09ae-479a-9a88-2b8a229bb724](https://user-images.githubusercontent.com/2582291/167375663-1d47b275-6499-4dae-acc8-5355a35ec59d.png)

Any idea why this happens?

**Comment by fmw42** [comment, marked as answer]
Your TIFF file has an alpha channel. Turn it off

`convert test.tiff -alpha off test2.pdf`

[test2.pdf](https://github.com/ImageMagick/ImageMagick/files/8652912/test2.pdf)
